articles/machine-learning/azure-machine-learning-release-notes-cli-v2.md

```diff
@@ -23,6 +23,37 @@ In this article, learn about Azure Machine Learning CLI (v2) releases.
 
 __RSS feed__: Get notified when this page is updated by copying and pasting the following URL into your feed reader:
 `https://learn.microsoft.com/api/search/rss?search=%22Azure+machine+learning+release+notes-v2%22&locale=en-us`
+## 2025-05-09
+
+## Azure Machine Learning CLI (v2) v 2.37.0
+- `az ml workspace create`
+  - Hub and Project workspace marked as GA.
+
+## 2025-04-23
+
+## Azure Machine Learning CLI (v2) v 2.36.5
+  - Pin major version of external dependencies in SDK.
+
+## 2025-04-23
```

Comment:
Can we remove the duplicate heading on line 37? Duplicate headings will possibly block publication of this.